index.spec.js

```diff
@@ -30,6 +32,15 @@ test('should render three button separated by a custom separator', () => {
 	expect(actual.text()).toEqual('One:Two:Three');
 });
 
+test('should render three button separated by react element <br/>', () => {
+	const actual = shallow(
+		createElement(Group, { separator: React.createElement('br') }, elements)
+	);
+
+	expect(console.error).not.toBeCalled();
```

Comment:
lol, i was having the same thoughts,

was looking at the documentation.
and was looking for something sinon-like restore.

i will change to the suggested way